4_basic_assembly/2_bitwise_operations/2_practical_bit_games/answers/3_write_code/0_0.asm

```diff
@@ -0,0 +1,37 @@
+; Basic Assembly
+; ==============
+; 
+; 0.0   Write a program that takes a number x as input, and returns:
+;       - 0 if x is even.
+;       - 1 if x is odd. 
+;
+; Answer: An integer is odd iff the LSB is 1.
+
+
+format PE console
+entry start
+
+include 'win32a.inc' 
+
+; ===============================================
+section '.text' code readable executable
+
+start:
+
+    call    read_hex
+    and     eax,1

```

Comment:
@liara : You can stop here with the calculation. At this point if eax was even, you have eax = 0. Otherwise you have eax = 1 (:

Reply:
damn, didn't see that:)